src/scripts/main.js

```diff
@@ -1,3 +1,23 @@
 'use strict';
 
-// write code here
+const treeItems = document.querySelectorAll('.tree li');
+
+treeItems.forEach((item) => {
+  const nestedList = item.querySelector('ul');
+
+  if (nestedList) {
+    const span = document.createElement('span');
+
+    span.textContent = item.childNodes[0].nodeValue.trim();
```

Comment:
There is a potential issue here. If the list item contains more than just text nodes (e.g., other elements or comments), `item.childNodes[0].nodeValue` might not return the expected text content. Consider using `item.firstChild` or `item.textContent` to ensure you are accessing the correct text node.